client/modules/IDE/reducers/files.js

```diff
@@ -13,15 +13,19 @@ const defaultHTML =
 `<!DOCTYPE html>
 <html>
   <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
why is this including jquery?

Reply:
we will clear this out - our library has dependencies on jquery but we will make them vanilla javascript.